trackhubs/tests/test_import_assemblies_command.py

```diff
@@ -0,0 +1,61 @@
+"""
+.. See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+
+import requests
+import pytest
+from trackhubs.management.commands.import_assemblies import import_ena_dump
+from trackhubs.models import GenomeAssemblyDump
+
+
+@pytest.mark.parametrize(
+    'test_url, expected_result',
+    [
+        ('https://www.ebi.ac.uk/ena/portal/api/search?format=json&limit=1&result=assembly', True),
+        ('https://api.genome.ucsc.edu/list/ucscGenomes', True)
+    ]
+)
+def test_request_response(test_url, expected_result):
```

Comment:
Hey @bilalebi,

What do you think of testing `fetch_assembly` method instead here?

Reply:
That's a good idea, I refactored `fetch_assembly()` to accept any URL and mocked the API's response